bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/AnomalousCouplings-SMP/ZZjj_llvvjj_ewk_aQGC/ZZjj_llvvjj_ewk_aQGC_reweight_card.dat

```diff
@@ -0,0 +1,13225 @@
+change helicity False
+change rwgt_dir rwgt
+
+launch --rwgt_name=sm
```

Comment:
Just out of curiosity, can you also check this sm-reweighted cross sections being comparable to the SM UFO?

Reply:
@sihyunjeon  thanks for the suggestion. I generated a gridpack using this datacard, and for SM standalone, I used the ZZ VBS one: `/cvmfs/cms.cern.ch/phys_generator/gridpacks/slc6_amd64_gcc630/13TeV/madgraph/V5_2.6.5/VBS_ZZ/ZZjj_llvvjj_ewk_slc6_amd64_gcc630_CMSSW_9_3_16_tarball.tar.xz`

For each gridpack, I submitted 200 jobs, with 1000 events in each job. The average and standard deviation of the cross-sections are:
xsec standalone mean: 0.003078 std: 0.000034
xsec reweighted mean: 0.003866 std: 0.000729

I've also attached their distributions. It seems the re-weighted SM has a much larger deviation than the standalone SM. Could this be an issue?
![newplot](https://user-images.githubusercontent.com/19845933/135643572-594ff6fb-b55f-4f82-aedb-64ab48f4511b.png)

Reply:
So my point was to set all the new physics couplings to 0 (which I expect to give you pure SM although you are using this UFO model) and then compare it to the default SM UFO model in MadGraph. And you did this but this is giving you the difference?